plugins/secretstores/docker/docker_test.go

```diff
@@ -83,17 +66,18 @@ func TestListGet(t *testing.T) {
 	keys, err := plugin.List()
 	require.NoError(t, err)
 	require.Len(t, keys, len(secrets))
-	for _, k := range keys {
-		_, found := secrets[k]
-		require.True(t, found)
+	// check if the returned array from List() is the same
+	// as the name of secret files
+	for secretFileName := range secrets {
+		require.Contains(t, keys, secretFileName)
 	}
 
 	// Get the secrets
 	for _, k := range keys {
 		value, err := plugin.Get(k)
 		require.NoError(t, err)
 		v, found := secrets[k]
-		require.True(t, found)
+		require.Truef(t, found, "unexpected secret requested that was not found: %q", secrets[k])
```

Comment:
Won’t this give another error as it can’t be found?

Reply:
@Hipska hmmm. That might be the case, should I revert back?

Reply:
You could just mention `k` is not found…